locales/en-US.json

```diff
@@ -30,6 +30,13 @@
   "darkModeToggle": "Toggle dark mode",
   "dateAdded": "Date added",
   "details": "Details",
+  "dialog": {
+    "upNext": {
+      "nextEpisodePlayingIn": "Next Episode Playing in",
```

Comment:
```suggestion
      "nextEpisodePlayingIn": "Next episode playing in",
```

Reply:
What's the reason for keeping everything capitalized? :)

Reply:
> What's the reason for keeping everything capitalized? :)

Weird English conventions that are not even really standard (It depends on the style guide used).

Reply:
> Weird English conventions that are not even really standard (It depends on the style guide used).
So is the consensus that it should be lowercased?

Reply:
"Next episode plays in..."